gradle-plugin/src/main/groovy/io/realm/gradle/Realm.groovy

```diff
@@ -39,19 +39,20 @@ class Realm implements Plugin<Project> {
             throw new GradleException('Realm gradle plugin only supports android gradle plugin 1.5.0 or later.')
         }
 
-        def isKotlinProject = project.plugins.find {
-            it.getClass().name == 'org.jetbrains.kotlin.gradle.plugin.KotlinAndroidPluginWrapper'
-        }
+        def hasKotlinPlugin = project.plugins.findPlugin('kotlin-android') != null
```

Comment:
I adopted `it.getClass().name == 'org.jetbrains.kotlin.gradle.plugin.KotlinAndroidPluginWrapper'` because some compatibility issue existed IIRC.

I'll check this.